providers/implementations/signature/rsa_sig.c

```diff
@@ -406,9 +409,6 @@ static int rsa_signverify_init(void *vprsactx, void *vrsa,
 
     prsactx->operation = operation;
 
-    if (!rsa_set_ctx_params(prsactx, params))
-        return 0;
-
     /* Maximum for sign, auto for verify */
     prsactx->saltlen = RSA_PSS_SALTLEN_AUTO;
     prsactx->min_saltlen = -1;
```

Comment:
Why do we need this block in *both* the newctx function and here?

Reply:
If we call the sign_init multiple times, it could be needed.